src/frontend_analysis/IR_analysis/InterfaceInfer.cpp

```diff
@@ -644,8 +653,12 @@ DesignFlowStep_Status InterfaceInfer::Exec()
                   {
                      continue;
                   }
-                  THROW_ERROR("Parameter '" + arg_name + "' cannot have interface type '" + interface_type +
-                              "' since no load/store is associated with it");
+                  // BEAWARE: none is used here in place of default to avoid memory allocation to consider this as
+                  // an active pointer parameter
+                  interface_type = "none";
```

Comment:
Not clear why memory allocation should bother with unused pointer parameters anyway.

Reply:
refactored the code to better manage unused parameters